app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt

```diff
@@ -216,7 +218,7 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
                 resources.getString(R.string.pocket_brand_name)) + " " + resources.getString(R.string.pocket_video_feed_reload_button)
 
         megaTileTryAgainButton.setOnClickListener { _ ->
-            pocketVideos = context!!.serviceLocator.pocket.getRecommendedVideos()
+            context!!.serviceLocator.pocketRepo.update()
```

Comment:
Did we decide as a team that we shouldn't go through the ViewModel to change these?

We should touch base with the team on our desired approach and be consistent. Maybe testability (e.g. swapping in a custom ViewModel during a UI test, we can have control over any side effects) is a concern.

Reply:
CC @psymoon ^ Since your implementation has these decisions too.

Reply:
No need for a meeting, this is just a mistake on my part.  I'll update to go through the VM.